Phonebook.Frontend/src/app/modules/table/components/table/table.component.ts

```diff
@@ -19,10 +20,14 @@ import { SearchState, SetTableResultCount, TableState, UpdateUrl } from 'src/app
   host: { class: 'pb-fill-parent' },
 })
 export class TableComponent implements OnInit, OnDestroy {
+  public sizeSmall: boolean = false;
```

Comment:
```suggestion
  public isSizeSmall: boolean = false;
```
(Optional) 
Also for other usages